packages/atomic/cypress/integration/facets/numeric-facet.cypress.ts

```diff
@@ -228,7 +225,7 @@ describe('Numeric with invalid options', () => {
           end: 100000,
         },
         {
-          start: 'x1000000',
+          start: ('x1000000' as unknown) as number,
```

Comment:
🤔

Reply:
> Should render a warning message when range start with non-number

Reply:
![Screen Shot 2021-03-31 at 9 33 24 AM](https://user-images.githubusercontent.com/4923043/113152458-2698ba80-9204-11eb-815a-d8eb1bcdc92a.png)
The interface itself was modified instead to allow it

Reply:
Oh :D